File: OCC/occ19b3.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        System.out.println((n % 4 == 0 || n % 4 == 3)? "Yes" : "No");
    }
}

File: OCC/occ19b3.go
package main
import "fmt"
func main(){
	var n int
	fmt.Scanf("%d", &n)
	if n % 4 == 0 || n % 4 == 3 {
		fmt.Println("Yes")
	}else{
		fmt.Println("No")
	}
}